config/initializers/s3.rb

```diff
@@ -6,9 +6,11 @@
 AWS_SECRET_ACCESS_KEY = ENV['AWS_SECRET_ACCESS_KEY']
 AWS_BUCKET            = ENV['AWS_BUCKET']
 AWS_REGION            = ENV['AWS_REGION']
+AWS_ENDPOINT_URL_S3   = ENV['AWS_ENDPOINT_URL_S3']
```

Comment:
Is this a standardized name? I'd rather use `AWS_ENDPOINT_URL` if not, since none of the other keys have a `_S3` suffix.

Reply:
Yes, that's standard. Removing S3 causes conflicts with other endpoints AWS provides

https://fly.io/docs/tigris/#create-and-manage-a-tigris-storage-bucket